build/ol/install-packages.sh

```diff
@@ -42,18 +46,25 @@ dnf install -y \
   findutils \
   which
 
+# Install Git LFS
 curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh | sudo bash
-dnf update -y
```

Comment:
This `dnf update -y` might still be missing

Reply:
Added back.